test_preparation.py

```diff
@@ -1,7 +1,33 @@
-""" unit test template """
+""" This is to test the functions created in the preparation module """
 
+import numpy as np
+import pandas as pd
+import pytest
 import preparation as prep
 
 def test_dummy():
```

Comment:
We can probably remove the dummy test function since we have actual code in this file now.